lib/src/core/util/ui_text.dart

```diff
@@ -15,11 +15,36 @@ abstract class UiText {
   static const String movieGenresLabel = 'Genres';
 
   //photoUrl
-  static const String movieURL= 'https://image.tmdb.org/t/p/w500/';
+  static const String movieURL = 'https://image.tmdb.org/t/p/w500/';
+
   //Overview
   static const String overviewLabel = 'Overview';
 
   //Home Screen
-  static const String homeScreenReleaseDate = 'Release date: ';
-  static const String homeScreenMovieTitle = 'Movie Title: ';
+  static const String homeScreenCategoryPopular = 'Popular';
+  static const String homeScreenCategoryNowPlaying = 'Now Playing';
+  static const String homeScreenCategoryTopRated = 'Top Rated';
+  static const String homeScreenCategoryUpcoming = 'Upcoming';
+
+  static const String homeScreenPlayNow = 'Play Now';
+
+  //API PROVIDER
+  static const String apiProviderDBUrl = 'http://api.themoviedb.org/3/movie/';
+  static const String apiProviderAPIKey =
+      '?api_key=447e725503a582e83ecf9a7a50bc9729';
+  static const String apiProviderGenresLink = 'https://api.themoviedb.org/3/genre/movie/list?language=en&api_key=447e725503a582e83ecf9a7a50bc9729';
```

Comment:
You create a const to api key and on apiProviderGenresLink you can use it instead to repeat it, like this:
`'https://api.themoviedb.org/3/genre/movie/list?language=en&api_key=$apiProviderAPIKey'`
And change apiProviderAPIKey from` '?api_key=447e725503a582e83ecf9a7a50bc9729'` to `'447e725503a582e83ecf9a7a50bc9729'`